### ISSUE TITLE
details question about attention map in Figure.3 (a)

### ISSUE THREAD

**Issue by dbdx-zwh** [beanie00/Decision-ConvFormer#2]
Congratulations for publishing your work in ICLR2024! I have seen your figure.3(a) said **original DT** cannot focus on local information. However, I run your released code with your env, and my attention heatmap in **DT** is similar to figure.3(b). Can you release more details (code is better) to reproduce the heatmap in you figure 3?

**Comment by beanie00** [comment]
Thank you for your interest in our work.

Regarding the attention map in Figure 3(a), we visualized it during the evaluation phase with a single batch size in the hopper-medium environment. Additionally, we used the default hyperparameters from https://github.com/kzl/decision-transformer (note that the default hyperparameters in our code differ slightly in terms of activation function and learning rate).
